src/summary.cpp

```diff
@@ -18,17 +18,18 @@
 //      
 
 
-#include "summary.h"
 #include <unistd.h>
+#include <klocalizedstring.h>
+#include "summary.h"
 
 Summary::Summary(PackageBackend *backend, QString *tmpFileName, QObject *parent )
 {
     m_backend = backend;
     m_tmpFileName = tmpFileName;
     m_installationSummary = new QTextBrowser;
     m_installationSummary->addScrollBarWidget( new QScrollBar, Qt::AlignRight );
-    m_continue = new QPushButton( "Continue Installation ");
-    m_cancel = new QPushButton( "Cancel ");
+    m_continue = new QPushButton( i18n("Continue Installation ") );
+    m_cancel = new QPushButton( i18n("Cancel ") );
```

Comment:
Why is there a space at the end of those two strings? I think that was wrong before too.